### DISCUSSION TITLE
Tipe Data

### DISCUSSION THREAD

**Discussion by ageljonathan** [teamalgoritma/community#447] [Q&A] [answered]
Maaf sblmnya saya ingin memastikan kembali
Untuk Logistic Regression variable prediktor lebih baik factor atau numerik?
Sedangkan untuk Regression Model lebih baik yg mana?
Terima kasih

**Comment by divakartika** [comment, marked as answer]
Dear @ageljonathan 

Untuk keduanya dapat digunakan prediktor factor maupun numerik. Terkait prediktor mana yang lebih baik, bergantung pada business case, variabel prediktor apa saja yang penting terhadap variabel target yang ingin diprediksi.

Semoga jawaban ini dapat membantu.

**Reply by ageljonathan** [reply]
Baik mbak
Terima kasih
